<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mine_degin_dlg</class>
 <widget class="QWidget" name="mine_degin_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>矿井设计</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../usecad.qrc">
    <normaloff>:/images/cbm.ico</normaloff>:/images/cbm.ico</iconset>
  </property>
  <widget class="QLabel" name="mineName_label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>市（地区）</string>
   </property>
  </widget>
  <widget class="QLabel" name="mineName_label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>省</string>
   </property>
  </widget>
  <widget class="QComboBox" name="region">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="base">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="province">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="city">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mineName_label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所在地</string>
   </property>
  </widget>
  <widget class="QLabel" name="mineName_label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所在矿区</string>
   </property>
  </widget>
  <widget class="QLabel" name="mineName_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>矿井名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="mineName_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>煤炭基地</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;输入矿井名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户名&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>水文地质条件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>地形地貌特征</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hydr_geo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>复杂</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>极复杂</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中等</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>简单</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="topo_geo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>复杂</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中等</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>简单</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="capacity">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mineName_label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>矿井产能</string>
   </property>
  </widget>
  <widget class="QLabel" name="mineName_label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mt/a</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ground_cond">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>是否具备地面井开发条件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>煤层列表</string>
   </property>
  </widget>
  <widget class="QComboBox" name="coal">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_coal">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_coal">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="more_coal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>详细设计</string>
   </property>
  </widget>
  <widget class="QComboBox" name="work_area">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>生产采区</string>
   </property>
  </widget>
  <widget class="QPushButton" name="more_work_area">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>详细设计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_work_area">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_work_area">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QComboBox" name="work_surf">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>回采工作面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="more_work_surf">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>详细设计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_work_surf">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_work_surf">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QComboBox" name="drilling_surf">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>掘进工作面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="more_rilling_surf">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>详细设计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_rilling_surf">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_drilling_surf">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../usecad.qrc"/>
 </resources>
 <connections/>
</ui>
